easybuild/easyconfigs/a/Anaconda/Anaconda-1.8.0-Linux-x86_64.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'Binary'
+
+name = 'Anaconda'
+version = '1.8.0-Linux-x86_64'
+
+homepage = 'https://store.continuum.io/cshop/anaconda/'
+description = """Completely free enterprise-ready Python distribution for large-scale data processing, predictive 
+ analytics, and scientific computing"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['https://repo.continuum.io/archive/']
+sources = ['%s-%s.sh' % (name, version)]
+
+qa = {
+            r'Please, press ENTER to continue': '\n',
+            r'Do you approve the license terms? [yes|no]': 'yes',
+            r'Or specify an different location below': '%(installdir)s/', 
+            r'Do you wish the installer to prepend the Anaconda install location to PATH': 'no',
```

Comment:
if you can maintain 4-space indentation, that helps lots of others of us follow the same style.